tests/integration/einstein/test_operations.py

```diff
@@ -222,3 +223,40 @@ def tearDown(self):
         self.db.project.delete_with_name(PROJECT)
         self.db.close()
         self.good_bmi.shutdown()
+
+
+class TestCopyImage(TestCase):
+    """
+    Creating a flatten copy of an image
+    """
+    @trace
+    def setUp(self):
+        self.db = Database()
+        self.db.project.insert(PROJECT, NETWORK)
+
+        self.good_bmi = BMI(CORRECT_HIL_USERNAME, CORRECT_HIL_PASSWORD,
+                            PROJECT)
+        self.good_bmi.import_ceph_image(EXIST_IMG_NAME)
+        img2 = uuid.uuid4()
+
+    def runTest(self):
+        response = self.good_bmi.copy_image(EXIST_IMG_NAME, PROJECT,
+                                            img2)
+        self.assertEqual(response[constants.STATUS_CODE_KEY], 200)
+
+        images = self.db.image.fetch_images_from_project(PROJECT)
+        exists_image = False
+        for image in images:
+            if img2 == image:
+                exists_image = True
+        self.assertTrue(exists_image)
+        time.sleep(constants.HIL_CALL_TIMEOUT)
```

Comment:
you can also remove this because we aren't provisioning. 

Provisioning and deprovisioning make calls to `hil` that can take some time, which is why we have that `sleep` call.